docs/ccdproc/image_management.rst

```diff
@@ -25,6 +25,10 @@ example initialization looks like::
     >>> keys = ['imagetyp', 'object', 'filter', 'exposure']
     >>> ic1 = ImageFileCollection('.', keywords=keys) # only keep track of keys
 
+    >>> # the following line shouldn't be needed in real files but it's needed
+    >>> # because the testfile has no header entry for "filter".
+    >>> ic1.summary['filter'] = ic1.summary['filter'].astype(str)
+
```

Comment:
You're right but there are a lot of places where inexistant header keywords (in particular `filter`) are used. I think that should be fixed in the complete test suite. That's a bit of effort, however requires very little knowledge about `ccdproc` itself so I figured it would make a good "first fix". The main objective of this PR is just to make the tests pass again.